website/versioned_docs/version-5.x/separating-applications.md

```diff
@@ -57,6 +61,7 @@ Changing which JavaScript file is loaded for each child application can be done
 
 |   | Separate code repositories possible | Independent CI builds | Separate deployments | Examples |
 | - | ----------------------------------- | --------------- | -------------------- | -------- |
-| Monorepo | :x: | :x: | :x: | &mdash; |
+| Monolith | :x: | :x: | :x: | &mdash; |
 | NPM Packages | :white_check_mark: | :white_check_mark: | :x: | [1](https://github.com/jualoppaz/single-spa-login-example-with-npm-packages) |
+| Monorepo | :white_check_mark: | :white_check_mark: | :white_check_mark: | &mdash; |
```

Comment:
I'm not against adding this row, but could you explain why you've checked "Separate code repositories possible" for monorepo? The definition of a monorepo is that there is a single code repository.

Additionally, I think we should add something besides a check mark for independent CI builds and separate deployments - perhaps an ❓ emoji, with a note about CircleCI? Since for the majority of CI tools that exist, this isn't possible.

Reply:
Sounds good to me. 

I know that Circle CI, Vercel and AWS Amplify CI all support mono-repos. Quite well supported now. With a `lerna` based mono-repo, all you need to do is call `lerna changed | grep $package_name` and it will return `true` if that package has changed. At work, we could easily add support for mono-repo on our Jenkins CI as well using this simple technique.

Reply:
How about this:

```suggestion
| Monorepo | :x: | :white_check_mark: [1](https://medium.com/labs42/monorepo-with-circleci-conditional-workflows-69e65d3f1bd0) | :white_check_mark: [1](https://medium.com/labs42/monorepo-with-circleci-conditional-workflows-69e65d3f1bd0) | &mdash; |
```

Reply:
Fine by me :)